src/sentry/utils/snuba.py

```diff
@@ -867,7 +872,7 @@ def get_cache_key(query: SnubaQuery) -> str:
 
 
 def _apply_cache_and_build_results(
```

Comment:
There is a unit test for this function that I suspected could fail after changing the input specs, but it doesnt seem that it did